### ISSUE TITLE
Youtube Singlevideo & Playlist Downloading 

### ISSUE THREAD

**Issue by Enes752** [JunkFood02/Seal#993]
### Checklist

- [ ] I'm reporting a bug unrelated to a specific site.
- [X] I've verified that I'm running the [**latest version**](https://github.com/yt-dlp/yt-dlp/releases/latest) of yt-dlp.
- [X] I've verified that I'm running the latest [**stable version**](https://github.com/JunkFood02/Seal/releases/latest/) of Seal or any later [**preview versions**](https://github.com/JunkFood02/Seal/releases).
- [X] I've read the [**Contributing guidelines**](https://github.com/JunkFood02/Seal/blob/main/CONTRIBUTING.md) and [**Code Of Conduct.**](https://github.com/JunkFood02/Seal/blob/main/CODE_OF_CONDUCT.md)
- [X] I've checkd that the site i'm trying to download from is in the [**Supported Sites**](https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md) list from yt-dlp
- [X] I understand that the issue will be (ignored/closed) if I intentionally remove or skip any mandatory field.

### Describe the bug

I can't download a video or an playlist from Youtube.

### To Reproduce

Copy a link from the error report and past on the download field and click on „download a video from playlist". Scroll down to „download" and press it and see the error, which I pasted to error reports.

### Error reports

App version: 1.10.0-beta.4 (10920)
Device information: Android 13 (API 33)
Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]
Yt-dlp version: 2023.07.06

URL: https://youtu.be/aIyS94s4aYs
WARNING: [youtube] unable to extract initial player response; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: [youtube] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: The download speed shown is only of one thread. This is a known issue
ERROR: 
[download] Got error: HTTP Error 403: Forbidden. Giving up after 10 retries
ERROR: fragment 1 not found, unable to continue
WARNING: The download speed shown is only of one thread. This is a known issue
ERROR: 
[download] Got error: HTTP Error 403: Forbidden. Giving up after 10 retries
ERROR: fragment 1 not found, unable to continue

### Device info

App version: 1.10.0-beta.4 (10920)
Device information: Android 13 (API 33)
Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]
Yt-dlp version: 2023.07.06

### Additional context

I got a issue for a downloading Youtube Video or an playlist. Can you please help me?

**Comment by Marco-9456** [comment]
Turn your DNS/VPN off if you use one and try again.

**Comment by Enes752** [comment]
> Turn your DNS/VPN off if you use one and try again.

The VPN or DNS is always deactivated. I have a problem with downloading.

**Comment by Enes752** [comment]
> https://github.com/yt-dlp/yt-dlp/issues/7811
> 
> TL;DR try long press yt-dlp version, switch to nightly build and update it

How can I long press it?

**Comment by Marco-9456** [comment]
@Enes752 

https://github.com/JunkFood02/Seal/assets/41828058/2b80b508-046c-48a2-aebe-e069c4b82016

**Comment by Enes752** [comment]
> @Enes752 
> 
> https://github.com/JunkFood02/Seal/assets/41828058/2b80b508-046c-48a2-aebe-e069c4b82016
> 
> 

Oh, thanks. I didn't noticed this situation.
